cmd/bacalhau/utils.go

```diff
@@ -189,6 +194,61 @@ func setupDownloadFlags(cmd *cobra.Command, settings *ipfs.IPFSDownloadSettings)
 		settings.IPFSSwarmAddrs, "Comma-separated list of IPFS nodes to connect to.")
 }
 
+//nolint
+func ConvertJobspecToFile(
```

Comment:
Delete all this. Please reuse the functionality in Describe to unmarshall the object into a YAML/JSON file. We don't need to support both - just use YAML.